dm/dm/worker/server.go

```diff
@@ -340,6 +340,10 @@ func (s *Server) observeRelayConfig(ctx context.Context, rev int64) error {
 					}
 					rev = rev1
 					if relaySource == nil {
+						if w := s.getWorker(false); w != nil && w.startedRelayBySourceCfg {
+							break
+						}
+						log.L().Info("didn't found relay config after etcd retryable error")
```

Comment:
Just suggestion
```suggestion
						log.L().Info("didn't found relay config after etcd retryable error. Will start to stop relay now")
```

Reply:
above comments should be fixed in https://github.com/pingcap/ticdc/pull/3190/commits/4d349c7e3dd1ec5592b55849e75a35a9ae8f7c77